extensions/positron-r/src/runtime-manager.ts

```diff
@@ -32,8 +35,20 @@ export class RRuntimeManager implements positron.LanguageRuntimeManager {
 	}
 
 	async recommendedWorkspaceRuntime(): Promise<positron.LanguageRuntimeMetadata | undefined> {
-		// TODO: If the workspace contains an R project, we could recommend an
-		// R runtime from e.g. the `DESCRIPTION` file or an renv lockfile.
+		// If the default interpreter path is set and the path exists on the filesystem,
+		// recommend it with implicit startup behavior.
+		const defaultInterpreterPath = getDefaultInterpreterPath();
+		if (defaultInterpreterPath) {
+			if (fs.existsSync(defaultInterpreterPath)) {
+				LOGGER.info(`[recommendedWorkspaceRuntime] Recommending R runtime from default interpreter path setting: ${defaultInterpreterPath}`);
```

Comment:
We could even give the actual name or key of the setting here, to help someone who is troubleshooting.